ui/load/ticket.go

```diff
@@ -0,0 +1,169 @@
+package load
+
+import (
+	"fmt"
+	"math"
+	"sort"
+	"time"
+
+	"github.com/decred/dcrd/dcrutil"
+	"github.com/planetdecred/dcrlibwallet"
+)
+
+type Ticket struct {
+	Status     string
+	Fee        string
+	Amount     string
+	DateTime   string
+	MonthDay   string
+	DaysBehind string
+	WalletName string
+
+	timestamp int64
+}
+
+const (
+	StakingLive     = "LIVE"
+	StakingImmature = "IMMATURE"
+	StakingExpired  = "EXPIRED"
+	StakingRevoked  = "REVOKED"
+	StakingVoted    = "VOTED"
+)
+
+func (wl *WalletLoad) StakingOverviewAllWallets() *dcrlibwallet.StakingOverview {
+	overview := new(dcrlibwallet.StakingOverview)
+	for _, w := range wl.MultiWallet.AllWallets() {
+		ov, _ := w.StakingOverview()
+		overview.All += ov.All
+		overview.Expired += ov.Expired
+		overview.Immature += ov.Immature
+		overview.Live += ov.Live
+		overview.Revoked += ov.Revoked
+		overview.Voted += ov.Voted
+	}
+
+	wl.MultiWallet.GetLowestBlockTimestamp()
+	return overview
+}
+
+func calculateDaysBehind(lastHeaderTime int64) string {
+	diff := time.Since(time.Unix(lastHeaderTime, 0))
+	daysBehind := int(math.Round(diff.Hours() / 24))
+	if daysBehind < 1 {
+		return "<1 day"
+	} else if daysBehind == 1 {
+		return "1 day"
+	} else {
+		return fmt.Sprintf("%d days", daysBehind)
+	}
+}
+
+func filterToStatus(txFilter int32) string {
```

Comment:
It looks like dcrlibwallet tx filters can be referenced directly instead of converting to identical constants.